origami.json

```diff
@@ -324,13 +324,13 @@
 			}
 		},
 		{
-			"title": "Hero Extra",
+			"title": "Hero Extra Centered",
```

Comment:
Ugh I can't comment on the lines above, I realise this isn't your doing, but it says Hero Centred ): could you chagne that too, pls 😬

Reply:
Would you prefer British or American spelling? The modifier class name is `o-teaser--centre` so "centred" everywhere?

Reply:
You know, I'm not sure, we use American for 'colors' in our docs. Let's go with British for now and I'll make a note of updating that documentation when I find it. 🙏